web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -17,12 +17,14 @@
   export let circle = false;
   export let hidden = false;
   export let border = false;
+  export let priority = false;
```

Comment:
Should this be defaulted to false? ImageThumbnail is used in a few different places.